src/components/templates/Poster/components/IntroVideoPreviewModal/IntroVideoPreviewModal.scss

```diff
@@ -0,0 +1,30 @@
+@import "scss/constants";
+
+$iframe-width: 50vw;
```

Comment:
Will the modal expand based on the content? I would expect we want to set the modal width to this, and then the iframe width to `100%`

Reply:
Changed in https://github.com/sparkletown/sparkle/pull/1299/commits/fef34d310713210aef00bbc6ad1367f651ef0fd8